AlfiyaSiddique/TastyTrails
# Add Delete Comment Functionality

## What does this PR do?
This PR implements the ability for users to delete comments on recipes. The following changes were made:
- Added a new handleDeleteComment function in the OneRecipe component.
- Integrated an API call to delete comments based on their ID.
- Updated the frontend to reflect the removal of a comment after it has been deleted.
- Ensured that only authorized users can delete comments by including a token in the request headers.

Fixes #261 


[Screencast from 2024-10-21 15-10-36.webm](https://github.com/user-attachments/assets/dc543764-699d-47f7-ae57-d6a1fa954f69)


## How should this be tested?




- Go to any recipe
- post any comment
- check that comment is being deleted or not

## Checklist



### Required
